### PR TITLE
Added 1.2.4 changes

### PR DESCRIPTION
**Pull request by MadelineMurray**
Signed-off-by: Madeline <madeline.murray@consensys.net>

Added 1.2.4 changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,29 @@ From v1.2, Besu requires Java 11.  Besu on Java 8 is no longer supported.
 In v1.2, we removed the entry-point script from our Docker image. Refer to the [migration guide](https://besu.hyperledger.org/en/latest/Deploying-Besu/High-Availability/)
 for information on options that were previously automatically added to the Besu command line. 
 
+## 1.2.4 
+
+### Additions and Improvements 
+
+- Add Istanbul block for Rinkeby [\#35](https://github.com/hyperledger/besu/pull/35)
```
Comment (timbeiko) [inline CHANGELOG.md]: `Add Istanbul fork block (5435345) for Rinkeby`

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,29 @@ From v1.2, Besu requires Java 11.  Besu on Java 8 is no longer supported.
...
+- Add Istanbul block for Goerli [\#27](https://github.com/hyperledger/besu/pull/27)
```
Comment (timbeiko) [inline CHANGELOG.md]: `Add Istanbul fork block (1561651) for Goerli`

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,29 @@ From v1.2, Besu requires Java 11.  Besu on Java 8 is no longer supported.
 In v1.2, we removed the entry-point script from our Docker image. Refer to the [migration guide](https://besu.hyperledger.org/en/latest/Deploying-Besu/High-Availability/)
 for information on options that were previously automatically added to the Besu command line. 
 
+## 1.2.4 
+
+### Additions and Improvements 
+
+- Add Istanbul block for Rinkeby [\#35](https://github.com/hyperledger/besu/pull/35)
+- Add Istanbul block for Goerli [\#27](https://github.com/hyperledger/besu/pull/27) 
+- Add Istanbul block for Ropsten [\#26](https://github.com/hyperledger/besu/pull/26)
```
Comment (timbeiko) [inline CHANGELOG.md]: Add Istanbul fork block (6485846) for Ropsten.

File: CHANGELOG.md (hunk 1 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 3 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Done

**Pull request merged**